django/db/backends/mysql/features.py

```diff
@@ -10,7 +10,13 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     # MySQL doesn't support sliced subqueries with IN/ALL/ANY/SOME.
     allow_sliced_subqueries_with_in = False
     has_select_for_update = True
-    has_select_for_update_nowait = False
+
+    # by default has_select_for_update_nowait and has_select_for_update_skip_locked are set to False
+    # no wait and skip locked features are introduced in MySQL V8.0.1
+    if version >= (8, 0, 1):
```

Comment:
This is the version of mysqlclient, not of MySQL. Look at `self.connection.mysql_version` in this file for the correct version.

Reply:
thanks, will update it tomorrow

Reply:
These also need changing to properties like `can_introspect_foreign_keys` because you can't inspect the database version at import time

Reply:
Thank you for your observation. I updated the pull request, could you check the new patch?